llvm/include/llvm/IR/Attributes.h

```diff
@@ -1498,10 +1498,10 @@ class CHERIoTGlobalCapabilityImportAttr {
       return false;
     }
 
-    Permissions = (HasRead ? std::string(ReadPermissionSymbol) : "") +
-                  (HasWrite ? std::string(WritePermissionSymbol) : "") +
-                  (HasCap ? std::string(CapPermissionSymbol) : "") +
-                  (HasMut ? std::string(MutPermissionSymbol) : "");
+    Permissions = (HasRead ? std::string(ReadPermissionSymbol) : "-") +
```

Comment:
I think this would be a good place to use `Twine(...)` over `std::string(...)`

See https://llvm.org/docs/ProgrammersManual.html#the-twine-class and https://llvm.org/docs/ProgrammersManual.html#dss-twine